.github/workflows/build.yaml

```diff
@@ -157,6 +143,20 @@ jobs:
           push: true
           tags: ghcr.io/castai/kvisor/kvisor-scanners:${{ env.head_commit_sha }},ghcr.io/castai/kvisor/kvisor-scanners:latest
 
+      - name: Run golangci-lint
```

Comment:
Moved lint and test to the last step because it takes some time but usually it's useful to get new docker image version for testing,